tests/acceptance/features/apiNotification/notification.feature

```diff
@@ -0,0 +1,242 @@
+        "messageRichParameters",
+        "notification_id",
+        "object_id",
+        "object_type",
+        "subject",
+        "subjectRich",
+        "user"
+      ],
+      "properties": {
+        "app": {
+          "type": "string",
+          "enum": ["userlog"]
+        },
+        "message": {
+          "type": "string",
+          "enum": ["Alice Hansen shared <resource> with you"]
+        },
+        "messageRich": {
+          "type": "string",
+          "enum": ["{user} shared {resource} with you"]
+        },
+        "messageRichParameters": {
+          "type": "object",
+          "required": [
+            "resource",
+            "user"
+          ],
+          "properties": {
+            "resource": {
+              "type": "object",
+              "required": [
+                "id",
+                "name"
+              ],
+              "properties": {
+                "id": {
+                  "type": "string",
+                  "pattern": "^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}\\$[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}![a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$"
+                },
+                "name": {
+                  "type": "string",
+                  "enum": ["<resource>"]
+                }
+              }
+            },
+            "user": {
+              "type": "object",
+              "required": [
+                "displayname",
+                "id",
+                "name"
+              ],
+              "properties": {
+                "displayname": {
+                  "type": "string",
+                  "enum": ["Alice Hansen"]
+                },
+                "id": {
+                  "type": "string",
+                  "enim": ["%user_id%"]
+                },
+                "name": {
+                  "type": "string",
+                  "enum": ["Alice"]
+                }
+              }
+            }
+          }
+        },
+        "notification_id": {
+          "type": "string"
+        },
+        "object_id": {
+          "type": "string"
+        },
+        "object_type": {
+          "type": "string",
+          "enum": ["share"]
+        },
+        "subject": {
+          "type": "string",
+          "enum": ["Resource shared"]
+        },
+        "subjectRich": {
+          "type": "string",
+          "enum": ["Resource shared"]
+        },
+        "user": {
+          "type": "string",
+          "enum": ["Alice"]
+        }
+      }
+    }
+    """
+    Examples:
+      | resource      |
+      | textfile1.txt |
+      | my_data       |
+
+
+  Scenario Outline: user gets a notification of unsharing resource
+    Given user "Alice" has shared entry "<resource>" with user "Brian"
+    And user "Brian" has accepted share "/<resource>" offered by user "Alice"
+    And user "Alice" has unshared folder "<resource>" form "Brian"
```

Comment:
```suggestion
    And user "Alice" has unshared entity "<resource>" from "Brian"
```

or may be something like this `And user "Alice" has unshared entity "<resource>" shared to "Brian"`